src/common/codeUtil.ts

```diff
@@ -7,227 +7,13 @@ export class CodeUtil {
         if (!original) {
```

Comment:
Let's keep the `test/suite/common/codeUtil.test.ts` and have it test the behavior of `CodeUtil.purify`. This function should have some clear tests as it's pretty core to the functionality of our extension :)

Reply:
Done.